test/index.js

```diff
@@ -2,619 +2,615 @@
   // indicating the function, and passing it an input
-  await alice.call("app", "holo_acorn", "create_whoami", {
+  await alice.call('app', 'holo_acorn', 'create_whoami', {
     profile: {
-      first_name: "alice",
-      last_name: "velandia",
-      status: "Online",
-      handle: "Branch",
-      avatar_url: "",
-      address: alice.info("app").agentAddress
+      first_name: 'alice',
+      last_name: 'velandia',
+      status: 'Online',
+      handle: 'Branch',
+      avatar_url: '',
+      address: alice.info('app').agentAddress
     }
-  });
-  await s.consistency();
-  const result2 = await alice.call("app", "holo_acorn", "create_whoami", {
+  })
+  await s.consistency()
+  const result2 = await alice.call('app', 'holo_acorn', 'create_whoami', {
     profile: {
-      first_name: "bob",
-      last_name: "romero",
-      handle: "Branch",
-      status: "Online",
-      avatar_url: "",
-      address: bob.info("app").agentAddress
+      first_name: 'bob',
+      last_name: 'romero',
+      handle: 'Branch',
+      status: 'Online',
+      avatar_url: '',
+      address: bob.info('app').agentAddress
     }
-  });
-  await s.consistency();
-  await bob.call("app", "holo_acorn", "create_whoami", {
+  })
+  await s.consistency()
+  await bob.call('app', 'holo_acorn', 'create_whoami', {
     profile: {
-      first_name: "bob",
-      last_name: "romero",
-      status: "Online",
-      handle: "Branch",
-      status: "Online",
-
-      avatar_url: "",
-      address: bob.info("app").agentAddress
+      first_name: 'bob',
+      last_name: 'romero',
+      status: 'Online',
+      handle: 'Branch',
+      status: 'Online',
+
+      avatar_url: '',
+      address: bob.info('app').agentAddress
     }
-  });
+  })
 
   // Wait for all network activity to
-  await s.consistency();
+  await s.consistency()
   const result_alice = await alice.call(
-    "app",
-    "holo_acorn",
-    "fetch_agent_address",
+    'app',
+    'holo_acorn',
+    'fetch_agent_address',
     {}
-  );
+  )
   const result_bob = await bob.call(
-    "app",
-    "holo_acorn",
-    "fetch_agent_address",
+    'app',
+    'holo_acorn',
+    'fetch_agent_address',
     {}
-  );
-  await s.consistency();
+  )
+  await s.consistency()
   // check for equality of the actual and expected results
-  const result = await alice.call("app", "holo_acorn", "fetch_agents", {});
+  const result = await alice.call('app', 'holo_acorn', 'fetch_agents', {})
 
-  t.equal(result.Ok.length, 2);
-  t.deepEqual(result2.Err, {
-    Internal:
-      '{"kind":{"ValidationFailed":"only the same agent as the profile is about can create their profile"},"file":"crates/core/src/nucleus/ribosome/runtime.rs","line":"220"}'
-  });
-  t.isNotDeepEqual(result_alice.Ok, result_bob.Ok);
-});
+  t.equal(result.Ok.length, 2)
+  const parsedError = JSON.parse(result2.Err.Internal)
+  t.equal(
+    parsedError.kind.ValidationFailed,
+    'only the same agent as the profile is about can create their profile'
```

Comment:
This is the meaningful change I made. I oopsied about committing style changes too.